docs/UserGuide.md

```diff
@@ -134,6 +135,7 @@ Adds a student to the address book.
 
 * New clashing schedule will be informed so that you can modify using the [`edit` command](#editing-a-student-edit).
 * <b>ADDRESS</b> can be used to store place of tuition. E.g. You can store tutee's address if the tuition happens at their place or you can store `My Place` if the tuition is at your place.   
+* For names containing special characters (e.g., `/`, `.`, or `&`), please omit these symbols and save the names in a memorable format, such as using only letters and spaces. This way, you can keep names easily identifiable within the app's constraints. 
```

Comment:
A suggestion, maybe the last sentence can be phrased to "This way, names can be easily identified by the app"?